### PR TITLE
Stop spewing stacks on error

### PR DESCRIPTION
**Pull request by mook-as**
[bsc#1076242](https://bugzilla.suse.com/show_bug.cgi?id=1076242): we were dumping the stack when things go wrong, because we were using the CF CLI terminal module (which upstream had noted at some point it was not designed for), and it was no longer quieting panics as expected.

This also includes changes to make this whole package build.

### CODE REVIEW

**Comment by andreas-kupries** [comment]
Fresh clone, first clone and checkout, then `make`:

Lint fail:

```

==>  Linting
Failed: /fast/go/src/github.com/SUSE/cf-usb-plugin/main.go:57:3: var usbUrl should be usbURL
Failed: /fast/go/src/github.com/SUSE/cf-usb-plugin/cmd/create-driver-endpoint.go:21:4: var targetUrl should be targetURL
Failed: /fast/go/src/github.com/SUSE/cf-usb-plugin/cmd/root.go:13:5: var configJson should be configJSON
Failed: /fast/go/src/github.com/SUSE/cf-usb-plugin/cmd/update-driver-endpoint.go:22:4: var targetUrl should be targetURL
Failed: /fast/go/src/github.com/SUSE/cf-usb-plugin/commands/instance.go:80:49: method parameter targetUrl should be targetURL
Failed: /fast/go/src/github.com/SUSE/cf-usb-plugin/commands/writer_unix.go:7:5: exported var Writer should have comment or be unexported
```

Test fail

```
go test -cover github.com/SUSE/cf-usb-plugin github.com/SUSE/cf-usb-plugin/cmd github.com/SUSE/cf-usb-plugin/commands github.com/SUSE/cf-usb-plugin/config github.com/SUSE/cf-usb-plugin/config/fakes github.com/SUSE/cf-usb-plugin/lib github.com/SUSE/cf-usb-plugin/lib/client/operations github.com/SUSE/cf-usb-plugin/lib/fakes github.com/SUSE/cf-usb-plugin/lib/models github.com/SUSE/cf-usb-plugin/lib/plugin
/fast/go/src/github.com/SUSE/cf-usb-plugin/vendor/github.com/SUSE/termui/error_printer.go:5:2: cannot find package "github.com/suse/termui/sigint" in any of:
	/fast/go/src/github.com/SUSE/cf-usb-plugin/vendor/github.com/suse/termui/sigint (vendor tree)
	/usr/local/go/src/github.com/suse/termui/sigint (from $GOROOT)
	/fast/go/src/github.com/suse/termui/sigint (from $GOPATH)
/fast/go/src/github.com/SUSE/cf-usb-plugin/vendor/github.com/SUSE/termui/ui.go:11:2: cannot find package "github.com/suse/termui/termpassword" in any of:
	/fast/go/src/github.com/SUSE/cf-usb-plugin/vendor/github.com/suse/termui/termpassword (vendor tree)
	/usr/local/go/src/github.com/suse/termui/termpassword (from $GOROOT)
	/fast/go/src/github.com/suse/termui/termpassword (from $GOPATH)
```

The test fail seems to be a `suse` vs `SUSE` mismatch in the go code.

**Comment by mook-as** [comment]
Linting issues are expected; I didn't want to fix them this round because the PR is already big enough and they don't break building.
The test error isn't though; do you happen to have a `github.com/suse` (lower case) in your `GOPATH`? I'll look on my side as well.

**Comment by mook-as** [comment]
Wait, that issue is in `termui` 😢 Let me make a PR against that…

**Comment by andreas-kupries** [comment]
Ok re linting.

Re the suse/SUSE I see

```
./vendor/github.com/SUSE/termui/termpassword/password_reader_unix.go:14:	"github.com/suse/termui/sigint"
./vendor/github.com/SUSE/termui/error_printer.go:5:	"github.com/suse/termui/sigint"
./vendor/github.com/SUSE/termui/ui.go:11:	"github.com/suse/termui/termpassword"
```

I.e. the vendored `termui` has bad include paths, expecting `suse` in two places.
However my GOPATH has only `SUSE`

```
andreask@ten:~/fast/dev/cf-usb-plugin> find /fast/go vendor -type d -print|grep -i '/suse$'
/fast/go/src/github.com/SUSE
vendor/github.com/SUSE
```

**Comment by andreas-kupries** [comment]
x-ed, confirming each other

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -13,6 +16,18 @@
 			"Comment": "v0.1.0-23-gd69616f",
 			"Rev": "d69616f51cdfcd7514d6a380847a152dfc2a749d"
 		},
+		{
+			"ImportPath": "github.com/SUSE/termui",
+			"Rev": "92556107bd806bea5d53728326e7d45c7b03f43c"
```
Comment (viovanov) [inline Godeps/Godeps.json]: This is still an old commit, how come?

**Review by viovanov — DISMISSED** [review]

File: Godeps/Godeps.json (hunk 1 above)
Comment (andreas-kupries) [reply Godeps/Godeps.json]: Mark did not update this yet. Likely waited for https://github.com/SUSE/termui/pull/8 to be merged.
AFAIK currently working on updating this.

**Review by andreas-kupries — APPROVED** [review]
Looks good now. Builds ok. Runs fine. No stacktraces for responding, but invalid endpoint

**Pull request merged**